scijava-ops-image/src/main/java/org/scijava/ops/image/simplify/RAIFocusers.java

```diff
@@ -0,0 +1,227 @@
+///*
+// * #%L
+// * ImageJ2 software for multidimensional image processing and analysis.
+// * %%
+// * Copyright (C) 2014 - 2023 ImageJ2 developers.
+// * %%
+// * Redistribution and use in source and binary forms, with or without
+// * modification, are permitted provided that the following conditions are met:
+// *
+// * 1. Redistributions of source code must retain the above copyright notice,
```

Comment:
I'd delete this

Reply:
Done! 6c9419aa8c9ad58f73e5e40d271c980cae2d1da8